<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat_room</class>
 <widget class="QWidget" name="Chat_room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_User">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>50</y>
     <width>20</width>
     <height>551</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="Chat_list">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>121</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>61</width>
     <height>181</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="add_new_btn">
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/add_new_user.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/add_new_user.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="User_net_btn">
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/user_net.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/user_net.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="renew_Btn">
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="text">
     <string>icon</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>781</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="chose_friend">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择一个聊天人</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="msg_text">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>781</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send_btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>590</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/send.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/send.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="Size_Btn">
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/bold.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/bold.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="I_Btn">
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/italic.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/italic.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="under_line_Btn">
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/under.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/under.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Color_Btn">
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/color.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/color.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="save_Btn">
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/save.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/save.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_Btn">
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/clear.jpg</normaloff>:/new/prefix1/C:/Users/30797/Desktop/icons_/clear.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>132</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>大小：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="size_Box"/>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="text">
     <string>字体：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox"/>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
